tests/local/network.go

```diff
@@ -33,14 +37,15 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ interfaces.LocalNetwork = &localNetwork{}
+var _ interfaces.LocalNetwork = &LocalNetwork{}
 
 // Implements Network, pointing to the network setup in local_network_setup.go
-type localNetwork struct {
-	teleporterContractAddress       common.Address
-	subnetAID, subnetBID, subnetCID ids.ID
-	subnetsInfo                     map[ids.ID]*interfaces.SubnetTestInfo
-	subnetNodeNames                 map[ids.ID][]string
+type LocalNetwork struct {
+	teleporterContractAddress common.Address
+	primaryNetworkInfo        *interfaces.SubnetTestInfo
```

Comment:
I think `cchainInfo`, etc is probably more accurate for the naming through out these tests, because the `SubnetTestInfo` doesn't have any value regarding the X or P chains.

Reply:
While true, I'd prefer to keep the naming consistent with respect to subnet vs blockchain for all subnets, including the primary network.

Reply:
Hm, fair point. The previous naming assumption was that a given subnet will only have 1 blockchain, which is still true for the subnets, but not true for primary network.

I'm fine with it for now, but would prefer we clean it up at some point...maybe `BlockchainTestInfo` for the struct name, and `subnetBlockchainInfo` rather than `subnetsInfo` (along with `cChainInfo`) to make everything fully accurate/consistent?

I'm also curious now if we need `subnetAID` and `subnetBID`, and if so if we should also add `primaryNetworkID` here, but can also be deferred to later clean up IMO.